### PR TITLE
Local Server Services 

### PR DESCRIPTION
**Pull request by nCastle1**
This sample demonstrates how to programmatically start and stop local server services, navigate to local server service URLs, and be notified of server and service state changes.

### CODE REVIEW

**Comment by nCastle1** [comment]
@ThadT Could you please give this a review?

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerServices/LocalServerServices.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+        private LocalGeoprocessingService _localGeoprocessingService;
+
+        // Generic reference to the service selected in the UI
+        private LocalService _selectedService;
+
+        public LocalServerServices()
+        {
+            InitializeComponent();
+
+            // set up the sample
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerServices/LocalServerServices.xaml.cs]: Capitalize first word of comment (for consistency).

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerServices/LocalServerServices.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+                btnServiceStart.IsEnabled = false;
+            }
+            else
+            {
+                btnServiceStop.IsEnabled = false;
+                btnServiceStart.IsEnabled = true;
+            }
+        }
+
+        private async void BtnStartServiceClick(object sender, RoutedEventArgs e)
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerServices/LocalServerServices.xaml.cs]: I don't like the Hungarian notation naming of controls <yuck>. That's just my opinion ;).

**Comment by nCastle1** [comment]
Thanks for the review! :)

**Pull request merged**